.gitignore

```diff
@@ -1,25 +1,46 @@
-/target/
+<<<<<<< HEAD
```

Comment:
Not really :-)

Reply:
OMG - I was worried about this. Fixing...